pkg/estimator/apis/types.go

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Please move to `pkg/apis/scheduler`.

Reply:
framework of scheduler is also defined in pkg/scheduler/apis/types.